components/crowdfunding-redesign/edit/EditFundraiser.tsx

```diff
@@ -0,0 +1,181 @@
+} from './common';
+
+const CoverImageForm = ({ initialValues, onSubmit }) => {
+  const tabs = React.useMemo(
+    () => [
+      { id: 'IMAGE', label: 'Upload photo' },
+      { id: 'VIDEO', label: 'Embed video' },
+    ],
+    [],
+  );
+
+  return (
+    <ColumnSection title="Cover photo" description="Add a cover photo to your fundraiser.">
+      <div className="flex flex-col gap-4">
+        <FormikZod schema={fundraiserSchema.pick({ cover: true })} initialValues={initialValues} onSubmit={onSubmit}>
+          {(formik: FormikProps<Fundraiser>) => {
+            return (
+              <Form>
+                <div className="flex flex-col items-start gap-4">
+                  <div className="w-fit">
+                    <Tabs
+                      selectedId={formik.values.cover.type}
+                      onChange={tab => formik.setFieldValue('cover.type', tab)}
+                      tabs={tabs}
+                    />
+                  </div>
+                  {formik.values.cover.type === 'IMAGE' ? (
+                    <FormField name="cover">
+                      {({ field }) => {
+                        const hasValidUrl = field.value && isValidUrl(field.value.url);
+
+                        return (
+                          <StyledDropzone
+                            name="cover.url"
+                            accept={DROPZONE_ACCEPT_IMAGES}
+                            minSize={10e2} // in bytes, =1kB
+                            maxSize={10e6} // in bytes, =10MB
+                            isMulti={false}
+                            showActions
+                            size={196}
+                            onSuccess={data => {
+                              formik.setFieldValue('cover.url', data.url);
+                            }}
+                            value={hasValidUrl && field.value.url}
+                          />
+                        );
+                      }}
+                    </FormField>
+                  ) : (
+                    <React.Fragment>
+                      <FormField name="cover.platform" label="Video platform">
+                        {({ field }) => {
+                          return (
+                            <Select
+                              {...field}
+                              onValueChange={platform => formik.setFieldValue('cover.platform', platform)}
+                              defaultValue="youtube"
+                            >
+                              <SelectTrigger>
+                                <SelectValue />
+                              </SelectTrigger>
+                              <SelectContent>
+                                {[{ label: 'Youtube', value: 'youtube' }].map(option => (
+                                  <SelectItem key={option.value} value={option.value}>
+                                    {option.label}
+                                  </SelectItem>
+                                ))}
```

Comment:
Platform can eventually be inferred from URL, see https://github.com/opencollective/opencollective-api/blob/e72d0a4e15578ea2c6304935c294a7d9206675f3/server/lib/notifications/utils.ts#L15

Reply:
Good point, right now I think there is some value in anchoring the user as to what video platforms we support